xml/System.Reflection/ReflectionTypeLoadException.xml

```diff
@@ -307,8 +307,8 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Returns the fully qualified name of this exception and the messages for all the loader exceptions.</summary>
+        <returns>A string containing the fully qualified name of this exception and the exception messages for the its loader exceptions.</returns>
```

Comment:
```suggestion
        <returns>A string containing the fully qualified name of this exception and the exception messages for its loader exceptions.</returns>
```

Reply:
Ah thanks for catching it. I thought this was going to be a flawless PR :(